pkg/template_generator.go

```diff
@@ -314,7 +384,16 @@ func addContainerValue(key string, s1 string, s2 string) string {
 }
 
 func addTemplateImageValue(containerName string, image string, key string, containerValue map[string]interface{}) string {
-	img := strings.Split(image, ":")
+	var img []string
+	var repo string = ""
+	if strings.Contains(image, "/") {
```

Comment:
@yadavnikhil, I think we should have 2 `values` here.
- Image Name: This can be any docker registry. eg, `appscode/voyager`, `quay.io/coreos/flannel`, `localhost.localdomain:5000/ubuntu` . This will allow Chart users to pull image from a different source. So, the logic in LN#410 needs to change.
- Image Tag

So, I think the logic should be,
```go
lastSlash := strings.LastIndex(image, '/')
lastColon := strings.LastIndex(image, ':')
if lastColon > lastSlash {
  // user provided image tag
} else {
  // user did not provide image tag, so use 'latest'
}
```